sorc/emcsfc_snow2mdl.fd/grib_utils.f

```diff
@@ -60,6 +98,7 @@ end subroutine grib_check
  SUBROUTINE SKGB2(LUGB,ISEEK,MSEEK,LSKIP,LGRIB,I1)
 !$$$  subprogram documentation block
 !
+!! @param[in] LUGB  input fortran unit default integer
```

Comment:
You must move the doxygen block above the subroutine statement and correct all the comment delimiters.